cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -109,9 +115,15 @@ func NewCreateCmd(clientGetter util.ClientGetter, streams genericiooptions.IOStr
 		Example:               createExample,
 		Args:                  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
 		ValidArgs:             []string{"job", "interactive"},
-		Run: func(cmd *cobra.Command, args []string) {
-			cobra.CheckErr(o.Complete(clientGetter, cmd, args))
-			cobra.CheckErr(o.Run(cmd.Context(), clientGetter))
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := o.Complete(clientGetter, cmd, args)
+			if err != nil {
+				return err
+			}
+			// stop the usage if we've got this far
+			cmd.SilenceUsage = true
```

Comment:
So on this case:

```suggestion
			// stop the usage if we've got this far
			cmd.SilenceUsage = true
			
			err := o.Complete(clientGetter, cmd, args)
			if err != nil {
				return err
			}
		
```

WDYT?

Reply:
done